api_emulator/redfish/EventListener_api.py

```diff
@@ -37,57 +44,58 @@ def getAggregationSource():
         if resource in agsource['Links']['ResourcesAccessed']:
             return agsource
 
+def createResource(redfish_obj):
+    obj_path = getRelativePath(redfish_obj)
+    file_path = create_path(constants.PATHS['Root'], obj_path)
+    create_object(redfish_obj, [], [], file_path)
 
+def fetchResource(obj_id, obj_root, host_url):
 
-class EventProcessor(Resource):
-    def __init__(self):
-        logging.info('Event Listener init called')
-        self.root = constants.PATHS['Root']
+    resource_endpoint = f"{host_url}/{obj_id}"
+    logging.info(f"fetch: {resource_endpoint}")
+    response = requests.get(resource_endpoint)
```

Comment:
This is breaking the unittests as it requires an agent to be up and running

Reply:
Good catch! I'll rebase and add the setup of an agent in the test pipeline